lib/PipelineC/PipelineC.cpp

```diff
@@ -29,7 +29,10 @@ void rp_set_custom_abort_hook(AbortHook Hook) {
   setAbortHook(Hook);
 }
 
-static char *copy_string(llvm::StringRef str) {
+/// Used when we want to return a stack allocated string.
+/// Copies the string onto the heap and gives ownership of
+/// to the caller
```

Comment:
This is not 80-cols aligned. Please make sure your editor is correctly set up, otherwise I always have to fix these.